perses/tests/test_geometry_engine.py

```diff
@@ -31,6 +31,18 @@
 
 from perses.rjmc.geometry import check_dimensionality
 from perses.utils.data import get_data_filename
+from perses.utils.openeye import smiles_to_oemol, OEMol_to_omm_ff
+from openmmforcefields.generators import SystemGenerator
+from openforcefield.topology import Molecule
+from simtk.openmm import app
+
+#global variables
+forcefield_files = ['amber14/protein.ff14SB.xml', 'amber14/tip3p.xml']
+small_molecule_forcefield = 'gaff-2.11'
+system_generator = SystemGenerator(forcefields = forcefield_files,
+                                                barostat = None,
+                                                forcefield_kwargs = { 'nonbondedMethod' : app.NoCutoff, 'implicitSolvent' : None, 'constraints' : None },
```

Comment:
Let's not specify things that we don't have to---`'implicitSolvent' : None` can be dropped, right?